src/panoptes/pocs/camera/camera.py

```diff
@@ -447,6 +473,24 @@ def take_exposure(self,
 
         assert filename is not None, self.logger.error("Must pass filename for take_exposure")
 
+        if not self.can_take_internal_darks:
+            if dark:
+                try:
+                    # Can't take internal dark, so try using an opaque filter in a filterwheel
+                    self.filterwheel.move_to_dark_position(blocking=True)
+                    msg = "Taking dark exposure using filter '" + \
+                          f"{self.filterwheel.filter_name(self.filterwheel._dark_position)}'."
+                    self.logger.debug(msg)
```

Comment:
Not sure how this `msg` syntax keeps slipping in ;)

```suggestion
                    self.logger.debug("Taking dark exposure using filter '"
                                      f"{self.filterwheel.filter_name(self.filterwheel._dark_position)}'.")
```

Reply:
Ah, `msg =` is 18 characters shorter than `self.logger.debug(`, which helps a bit with getting these big format strings within the line limits with fewer line breaks...